db/write_thread.h

```diff
@@ -25,9 +26,29 @@
 #include "rocksdb/types.h"
 #include "rocksdb/write_batch.h"
 #include "util/autovector.h"
-#include "util/safe_queue.h"
 
 namespace rocksdb {
+struct CommitRequest {
+  uint64_t commit_lsn;
+  // We use finished_write to check whether this writer has applied to memtable.
+  std::atomic<bool> finished_write;
+  std::atomic<bool> committed;
```

Comment:
I don't see this is getting accessed outside the mutex, why atomic?

Reply:
Good catch....  I judged whether the task may be committed outside the mutex before , but now it seems to be unnecessary...

Reply:
done